modules/packages/LinearAlgebra.chpl

```diff
@@ -503,8 +503,44 @@ proc transpose(A: [?Dom] ?eltType) where isDenseMatrix(A) {
   }
 }
 
+pragma "no doc"
+proc tranpose(A: [?Dom] ?eltType, keepDistribution = true) 
```

Comment:
Maybe `transposeDistribution` or `transposeDist` is a better name for this flag? Obivously, the default would be `false` in that case.

That argument can also be made `param`, but I do not have a strong preference one way or another.